### ISSUE TITLE
Can't start hyperopt

### ISSUE THREAD

**Issue by povstyany** [freqtrade/freqtrade#1270]
## Step 1: Have you search for this issue before posting it?

If you have discovered a bug in the bot, please [search our issue tracker](https://github.com/freqtrade/freqtrade/issues?q=is%3Aissue). 
If it hasn't been reported, please create a new issue.

## Step 2: Describe your environment

Windows withour docker or WSL.  Standart environment according to installation instuctions
  
## Step 3: Describe the problem:

I can't run hyperopt, always get error:
AttributeError: Can't get attribute 'DefaultStrategy' on <module 'freqtrade.strategy' 

### Steps to reproduce:

  1. Pull repo
  2. Install freqtrade according to instructions
  3. Run: python3 ./freqtrade/main.py -c config.json hyperopt -e 5000
  
### Observed Results:

I alway get the error and can' t exit of executing. Ctrl+C shows other error and the only way to restart the operation is to reopen the cmd

### Relevant code exceptions or logs:

  ```
ing.py:29: DeprecationWarning: numpy.core.umath_tests is an internal NumPy module and should not be imported. It will be removed in a future NumPy release.
  from numpy.core.umath_tests import inner1d
2018-10-09 22:48:20,423 - freqtrade.configuration - INFO - Verbosity set to 0
2018-10-09 22:48:20,424 - freqtrade.configuration - INFO - Dry run is enabled
2018-10-09 22:48:20,424 - freqtrade.configuration - INFO - Using DB: "sqlite://"
2018-10-09 22:48:20,424 - freqtrade.configuration - INFO - Using max_open_trades: 3 ...
2018-10-09 22:48:20,425 - freqtrade.configuration - INFO - Using data folder: user_data\data\bittrex ...
2018-10-09 22:48:20,426 - freqtrade.configuration - INFO - Parameter --epochs detected ...
2018-10-09 22:48:20,426 - freqtrade.configuration - INFO - Will run Hyperopt with for 5 epochs ...
2018-10-09 22:48:20,426 - freqtrade.configuration - INFO - Parameter -s/--spaces detected: all
2018-10-09 22:48:20,429 - freqtrade.strategy.resolver - INFO - Using resolved strategy DefaultStrategy from 'c:\favorites\php_development\openserver\domains\freqtrade\freqtrade\freqtrade\strategy'
2018-10-09 22:48:20,430 - freqtrade.strategy.resolver - INFO - Override strategy 'ticker_interval' with value in config file: 5m.
2018-10-09 22:48:20,432 - freqtrade.strategy.resolver - INFO - Using resolved strategy DefaultStrategy from 'c:\favorites\php_development\openserver\domains\freqtrade\freqtrade\freqtrade\strategy'
2018-10-09 22:48:20,433 - freqtrade.strategy.resolver - INFO - Override strategy 'minimal_roi' with value in config file: {'40': 0.0, '30': 0.01, '20': 0.02, '0': 0.04}.
2018-10-09 22:48:20,434 - freqtrade.strategy.resolver - INFO - Override strategy 'stoploss' with value in config file: -0.1.
2018-10-09 22:48:20,435 - freqtrade.strategy.resolver - INFO - Override strategy 'ticker_interval' with value in config file: 5m.
2018-10-09 22:48:20,435 - freqtrade.strategy.resolver - INFO - Override process_only_new_candles 'process_only_new_candles' with value in config file: False.
2018-10-09 22:48:20,436 - freqtrade.exchange - INFO - Instance is running with dry_run enabled
2018-10-09 22:48:21,074 - freqtrade.exchange - INFO - Using Exchange "Bittrex"
2018-10-09 22:48:23,663 - freqtrade.optimize - WARNING - No data for pair: "BTC/USD", Interval: 5m. Use --refresh-pairs-cached to download the data
2018-10-09 22:48:23,667 - freqtrade.optimize.hyperopt - INFO - Found 4 CPU cores. Let's make them scream!
C:\Favorites\PHP_development\OpenServer\domains\freqTrade\freqtrade\.env\lib\site-packages\sklearn\ensemble\weight_boosting.py:29: DeprecationWarning: numpy.core.umath_tests is an internal NumPy module and should not be imported. It will be removed in a future NumPy release.
  from numpy.core.umath_tests import inner1d
Process SpawnPoolWorker-4:
Traceback (most recent call last):
  File "C:\Program Files (x86)\Python36\lib\multiprocessing\process.py", line 258, in _bootstrap
    self.run()
  File "C:\Program Files (x86)\Python36\lib\multiprocessing\process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Program Files (x86)\Python36\lib\multiprocessing\pool.py", line 108, in worker
    task = get()
  File "C:\Favorites\PHP_development\OpenServer\domains\freqTrade\freqtrade\.env\lib\site-packages\sklearn\externals\joblib\pool.py", line 362, in get
    return recv()
  File "C:\Program Files (x86)\Python36\lib\multiprocessing\connection.py", line 251, in recv
    return _ForkingPickler.loads(buf.getbuffer())
AttributeError: Can't get attribute 'DefaultStrategy' on <module 'freqtrade.strategy' from 'c:\\favorites\\php_development\\openserver\\domains\\freqtrade\\freqtrade\\freqtrade\\strategy\\__init__.py'>
C:\Favorites\PHP_development\OpenServer\domains\freqTrade\freqtrade\.env\lib\site-packages\sklearn\ensemble\weight_boosting.py:29: DeprecationWarning: numpy.core.umath_tests is an internal NumPy module and should not be imported. It will be removed in a future NumPy release.
  from numpy.core.umath_tests import inner1d
Process SpawnPoolWorker-3:
Traceback (most recent call last):
  File "C:\Program Files (x86)\Python36\lib\multiprocessing\process.py", line 258, in _bootstrap
    self.run()
  File "C:\Program Files (x86)\Python36\lib\multiprocessing\process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Program Files (x86)\Python36\lib\multiprocessing\pool.py", line 108, in worker
    task = get()
  File "C:\Favorites\PHP_development\OpenServer\domains\freqTrade\freqtrade\.env\lib\site-packages\sklearn\externals\joblib\pool.py", line 362, in get
    return recv()
  File "C:\Program Files (x86)\Python36\lib\multiprocessing\connection.py", line 251, in recv
    return _ForkingPickler.loads(buf.getbuffer())
AttributeError: Can't get attribute 'DefaultStrategy' on <module 'freqtrade.strategy' from 'c:\\favorites\\php_development\\openserver\\domains\\freqtrade\\freqtrade\\freqtrade\\strategy\\__init__.py'>
C:\Favorites\PHP_development\OpenServer\domains\freqTrade\freqtrade\.env\lib\site-packages\sklearn\ensemble\weight_boosting.py:29: DeprecationWarning: numpy.core.umath_tests is an internal NumPy module and should not be imported. It will be removed in a future NumPy release.
  from numpy.core.umath_tests import inner1d
Process SpawnPoolWorker-1:
Traceback (most recent call last):
  File "C:\Program Files (x86)\Python36\lib\multiprocessing\process.py", line 258, in _bootstrap
    self.run()
  File "C:\Program Files (x86)\Python36\lib\multiprocessing\process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Program Files (x86)\Python36\lib\multiprocessing\pool.py", line 108, in worker
    task = get()
  File "C:\Favorites\PHP_development\OpenServer\domains\freqTrade\freqtrade\.env\lib\site-packages\sklearn\externals\joblib\pool.py", line 362, in get
    return recv()
  File "C:\Program Files (x86)\Python36\lib\multiprocessing\connection.py", line 251, in recv
    return _ForkingPickler.loads(buf.getbuffer())
AttributeError: Can't get attribute 'DefaultStrategy' on <module 'freqtrade.strategy' from 'c:\\favorites\\php_development\\openserver\\domains\\freqtrade\\freqtrade\\freqtrade\\strategy\\__init__.py'>
C:\Favorites\PHP_development\OpenServer\domains\freqTrade\freqtrade\.env\lib\site-packages\sklearn\ensemble\weight_boosting.py:29: DeprecationWarning: numpy.core.umath_tests is an internal NumPy module and should not be imported. It will be removed in a future NumPy release.
  from numpy.core.umath_tests import inner1d
Process SpawnPoolWorker-2:
Traceback (most recent call last):
  File "C:\Program Files (x86)\Python36\lib\multiprocessing\process.py", line 258, in _bootstrap
    self.run()
  File "C:\Program Files (x86)\Python36\lib\multiprocessing\process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Program Files (x86)\Python36\lib\multiprocessing\pool.py", line 108, in worker
    task = get()
  File "C:\Favorites\PHP_development\OpenServer\domains\freqTrade\freqtrade\.env\lib\site-packages\sklearn\externals\joblib\pool.py", line 362, in get
    return recv()
  File "C:\Program Files (x86)\Python36\lib\multiprocessing\connection.py", line 251, in recv
    return _ForkingPickler.loads(buf.getbuffer())
AttributeError: Can't get attribute 'DefaultStrategy' on <module 'freqtrade.strategy' from 
  ```

**Comment by xmatthias** [comment]
Hi, 
please add share exact version (`git log --format="%H" -n 1`)

in addition to that - we would need the version of `scikit-learn` as i saw this error related to a version of this module recently. 

in powershell, you should get that using `pip freeze | select-string scikit` - not sure about bash, eventually post the full output of `pip freeze -l` (with the virtualenvironment activated).

**Comment by povstyany** [comment]
Hi, thank you for answer. Here is info:

Exact version:
89b515be606edcb7db6804f23afa1c733c280a44

Version of scikit:
scikit-learn==0.19.2
scikit-optimize==0.5.2

**Comment by xmatthias** [comment]
can you please update the bot and retry?

this problem should be solved in #1247 - which was merged 3 days after the commit you refer to above

**Comment by povstyany** [comment]
Looks like it works. Thank you!
